README.md

```diff
@@ -16,6 +16,21 @@ Parameters in some Making Tax Digital for Income Tax APIs map to box numbers in
 
 **Note:** The date shown is the date that the change was released to Sandbox or Production.
 
+---
+
+### 3 September 2024
+#### Individuals Charges API
+
+The following change is now available in Sandbox for v2.0.
+
+##### Fixed
+
+Create and Amend Pension Charges:
+- `pensionSavingsTaxCharges` object in the request body can no longer be submitted for tax years 2024-25 onwards, and error `RULE_INCORRECT_OR_EMPTY_BODY_SUBMITTED` is returned when requests for tax years 2024-25 onwards include this object.
+
+Retrieve Pension Charges:
+- `pensionSavingsTaxCharges` object in the request body can no longer be submitted for tax years 2024-25 onwards, and error `RULE_INCORRECT_OR_EMPTY_BODY_SUBMITTED` is returned when requests for tax years 2024-25 onwards include this object.
+
```

Comment:
Can you please change the text for to Retrieve Pension Charges:- `pensionSavingsTaxCharges` object can no longer be retrieved in the response body for tax years 2024-25 onwards.